aria_data_loaders/aria_data_utils/conversions.py

```diff
@@ -0,0 +1,241 @@
+from typing import Any, Dict, List, Optional, Tuple
```

Comment:
renaming following convention we set up in previous PR #106

Reply:
addressed